ddtrace/contrib/_trace_utils_llm.py

```diff
@@ -24,7 +24,7 @@
 class BaseLLMIntegration:
     _integration_name = "baseLLM"
 
-    def __init__(self, config, stats_url, site, api_key):
+    def __init__(self, config, stats_url, site, api_key, app_key=None):
```

Comment:
Should app_key be an optional argument?  Do we need to raise an errorr/log warning if it is None

Reply:
`app_key` doesn't always need to be provided if `llmobs_enabled` is not set to true, in that case we raise an error on startup. I'll make the error more specific to show how to disable this feature.